spec/features/phone/add_phone_spec.rb

```diff
@@ -37,7 +41,9 @@
   scenario 'adding a new phone number validates number', js: true do
     user = create(:user, :signed_up)
     sign_in_and_2fa_user(user)
-    click_on "+ #{t('account.index.phone_add')}"
+    within('.sidenav') do
```

Comment:
I think the reason this is failing is because when `js: true` is enabled for a test, it's running a real browser and checking for real visibility of the link. Since that browser is sized at 800x600†, it's using the mobile appearance, with the link tucked away in the "Menu" button flyout navigation. A few thoughts for how we could address this:

- Change the default size of the Chrome window in [the browser configuration](https://github.com/18F/identity-idp/blob/effb3eb3531ae60c98bb466c155b5bea13eb054e/spec/support/capybara.rb#L7-L18)
   - I notice we [already do this for the smoke tests](https://github.com/18F/identity-idp/blob/effb3eb3531ae60c98bb466c155b5bea13eb054e/spec/monitor_spec_helper.rb#L13)
- Change the logic here to match the mobile experience, i.e. "click the 'Menu' button", then within the mobile navigation, click 'Add phone number'"


---

† Checked with:

```
pry(#<RSpec::ExampleGroups::AddANewPhoneNumber>)> Capybara.current_session.current_window.size
=> [800, 600]
```

Reply:
Thanks for the tip @aduth I'll look at this now.